Tribler/Core/Modules/restapi/channels/channels_subscription_endpoint.py

```diff
@@ -84,10 +84,8 @@ def render_PUT(self, request):
         """
         request.setHeader('Content-Type', 'text/json')
         channel_info = self.get_channel_from_db(self.cid)
-        if channel_info is None:
-            return ChannelsModifySubscriptionEndpoint.return_404(request)
 
-        if channel_info[7] == VOTE_SUBSCRIBE:
+        if channel_info is not None and channel_info[7] == VOTE_SUBSCRIBE:
```

Comment:
So if the channel cannot be found (is `None`) then you have already subscribed to it?